### PR TITLE
docs(samples): storage get bucket class and location

### PR DESCRIPTION
**Pull request by athakor**

### CODE REVIEW

File: google-cloud-examples/src/main/java/com/google/cloud/examples/storage/buckets/GetBucketStorageClassAndLocation.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import com.google.cloud.storage.StorageOptions;
+
+public class GetBucketStorageClassAndLocation {
+  public static void getBucketStorageClassAndLocation(String projectId, String bucketName) {
+    // The ID of your GCP project
+    // String projectId = "your-project-id";
+
+    // The ID to give your GCS bucket
+    // String bucketName = "your-unique-bucket-name";
+
```
Comment (lesv) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/storage/buckets/GetBucketStorageClassAndLocation.java:32]: See the [Sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md), about something.to call this method.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/storage/buckets/GetBucketStorageClassAndLocation.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    Storage storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
+
+    // See the StorageClass documentation for other valid storage classes:
+    // https://googleapis.dev/java/google-cloud-clients/latest/com/google/cloud/storage/StorageClass.html
+    StorageClass storageClass = StorageClass.COLDLINE;
+
+    // See this documentation for other valid locations:
+    // http://g.co/cloud/storage/docs/bucket-locations#location-mr
+    String location = "ASIA";
```
Comment (lesv) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/storage/buckets/GetBucketStorageClassAndLocation.java:41]: Might you wish to include this as argus passed into the function?

Comment (athakor) [reply google-cloud-examples/src/main/java/com/google/cloud/examples/storage/buckets/GetBucketStorageClassAndLocation.java:39]: done

**Comment by athakor** [comment]
@lesv both the comments have been addressed. PTAL

**Comment by lesv** [comment]
why was this closed?

**Comment by AVaksman** [comment]
It seems that it's already supported in the displaying a bucket's metadata sample here
https://cloud.google.com/storage/docs/getting-bucket-information#display-bucket-properties

**Pull request closed without merging**